### ISSUE TITLE
Snippet Selection: snippetType parameter is ignored

### ISSUE THREAD

**Issue by AndreasA** [sulu/sulu#4298]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 2.0-dev (930731896827d00fdc21d884327dc84256868c94)
| Browser Version | Chrome 70

#### Actual Behavior

When specifying the parameter snippetType for snippet selection type in the page template XML, it is completely ignored when trying to add the snippet in the backend.

#### Expected Behavior

Only allow specified snippet types.

#### Steps to Reproduce

1. Add a snippet definition
1. Add a second snippet definition
1. Create snippets for both in the admin interface
1. Add a snippet selection to the page template XML and only allow one of those snippet types by specifying the parameter.
1. Try to add a snippet in the admin interface and it will show both.

**Comment by danrot** [comment]
Thanks for the bug report! We are going to fix that before the final 2.0 release.

**Comment by nnatter** [comment]
Just tested this with Sulu 2.1.2 - when using the following property, only snippets of the type `other` can be selected in the administration interface:

```xml
<property name="snippet_selection" type="snippet_selection">
    <meta>
        <title lang="en">Snippet Selection</title>
    </meta>

    <params>
        <param name="types" value="other"/>
    </params>
</property>
```

Thanks for reporting again!
